src/lib/json.d

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+
+    Various helper functions
+
+    Copyright:
+        Copyright (c) 2017 sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module lib.json;
+
+import vibe.data.json;
+
+/*******************************************************************************
+
+    Short-hand access function for Json objects.
+    Instead of writiting json["abc"]["efg"]["hij"] you can do
+    json.path!"abc.efg.hij".
+
+    Params:
+        path_str = string making up the path to access the field
+        obj = object to access on
+
+    Returns:
+        json object at the given path
+
+*******************************************************************************/
+
+public Json path ( string path_str ) ( Json obj )
```

Comment:
Why template argument?

```D
public Json path ( Json obj, string path_str  )
{
    Json ret = obj;
    foreach (key; path_str.splitter('.'))
        ret = ret[key];
    return ret;
}
````

Reply:
Mind you, i just moved the function in this commit.

About the question.. I mainly thought "why not". I generally thought calculating at compile time is preferable to calculating at run time.

Reply:
> @mathias-baumann-sociomantic this is ready to review/merge, right?

Yes